cluster/examples/kubernetes/ceph/operator-openshift.yaml

```
@@ -356,6 +356,9 @@ data:
   # Whether to start the discovery daemon to watch for raw storage devices on nodes in the cluster.
   # This daemon does not need to run if you are only going to create your OSDs based on StorageClassDeviceSets with PVCs.
   ROOK_ENABLE_DISCOVERY_DAEMON: "false"
+  # Enable volume replication controller
+  CSI_ENABLE_VOLUME_REPLICATION: "true"
```

We're disabling by default, right?
```suggestion
  CSI_ENABLE_VOLUME_REPLICATION: "false"
```

---

Yes! But I was under the impression we should enable it for testing purposes.

---

For testing purposes, instead of setting it in operator.yaml, we need to replace it in the test settings. See [replaceOperatorSettings()](https://github.com/rook/rook/blob/fcf634e3b7808e4338e7fd261c047aae4793e99b/tests/framework/installer/ceph_settings.go#L103-L109)